include/libdnf-cli/output/transaction_table.hpp

```diff
@@ -74,7 +74,8 @@ class ActionHeaderPrinter {
     // ActionHeaderPrinter<Transaction::TransactionItem> action_header_printer(...);
     template <class T>
     struct libscols_line * print(const T & tspkg) {
```

Comment:
Please can you make a commit message shorter - Dropping the prefix will be quite helpful here.